feg/gateway/services/s6a_proxy/servicers/ul.go

```diff
@@ -55,6 +64,27 @@ func (s *s6aProxy) sendULR(sid string, req *protos.UpdateLocationRequest, retryC
 	return err
 }
 
+// createULR_Flags creates ULR Flags based on TS 29.272
+func createULR_Flags(req *protos.UpdateLocationRequest) int {
+	// ULR_FLAGS contains defaults flags for ULR-Flags
+	ulrFlags := ULR_FLAGS
+	if req.DualRegistration_5GIndicator {
+		// add TS 29.272 Dual-Registration5G-Indicator on bit 8
+		ulrFlags = ulrFlags | 1<<8
+	}
+	return ulrFlags
+}
+
+// createFeatureListID2 creates the Feature-List-ID 2 based on TS 29.272
+func createFeatureListID2(id2 *protos.FeatureListId2) int {
+	res := 0
+	if id2 != nil && id2.NrAsSecondaryRat {
+		// set bit 27 (TS 29.272 Nr As Secondary Rat)
+		res = res | 1<<27
+	}
+	return res
+}
+
```

Comment:
maybe: 
  
```
  func createFeatureListID2(id2 *protos.FeatureListId2) (ret int) {
	if id2 == nil { return }
	switch {
         case id2.NrAsSecondaryRat:
		// set bit 27 (TS 29.272 Nr As Secondary Rat)
		res = res | Bit27 // or whatever the appropriate name would be
        // // some future case -
        // case id2.SomeFutureFlag:
        //       res = res | SomeFutureBit
         }
         return
  }

```

Reply:
Ok with that example, now I get your other comment with the `type URLFlag int`

And it looks nice that way. Will adopt it